scripts/generate_test_suites.py

```diff
@@ -0,0 +1,39 @@
+import os
+import sys
+
+parts = sys.argv[1]
+
+instrumentation_tests_path = 'instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests'
+core_tests = os.listdir(f'{instrumentation_tests_path}/core')
```

Comment:
So we'll have to change something manually only when we add some package except for core/ui, right? Do we have a check that would make sure we don't forget about it?

Reply:
I think we should not divide dynamic suites by packages, only by part size. I need to refactor it